README.md

```diff
@@ -77,6 +77,9 @@ Visual Studio Codespaces is the easiest way to evaluate helium. Follow the setup
 # set environment variables
 export AUTH_TYPE=CLI
 
+# replace <your_key_vault_name> with the name of your key vault; keep the double quotes
```

Comment:
shouldn't this already be set to He_Name in most cases? Should we call that out?

we actually set auth_type and keyvault_name in saveenv.sh - we should call that out